hs_collection_resource/views.py

```diff
@@ -162,33 +148,33 @@ def calculate_collection_coverages(request, shortkey, *args, **kwargs):
 
 def _update_collection_coverages(collection_res_obj):
     """
-    update the collection coverages metadata.
-    :param collection_res_obj: collection obj
-    :return: a json response contains a list of coverage metadata dict
+    Update the collection coverages metadata records in db.
+    This func always removes all existing coverage metadata instances first,
+    and then create new ones if needed.
+    The element id of new created coverage metadata instance is stored in key "element_id_str".
+    :param collection_res_obj: instance of CollectionResource type
+    :return: a list of coverage metadata dict
     """
-    res_id = collection_res_obj.short_id
     new_coverage_list = _calculate_collection_coverages(collection_res_obj)
     try:
```

Comment:
This ```try``` here is not needed since on exception nothing meaningful work is being done. The function that calls this function already has a ```try```.